azure-pipelines.yml

```diff
@@ -86,11 +86,11 @@ stages:
   #                                                  Signed build                                                     #
   #-------------------------------------------------------------------------------------------------------------------#
   - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.9') }}:
+    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.8') }}:
```

Comment:
Also should be 17.9

Reply:
I accidentally reverted this file to an outdated `main` I had locally. But I already fixed it, there's no changes to the azure-pipelines in the PR anymore.